terracumber_config/tf_files/SUSEManager-4.2-PRV.tf

```diff
@@ -0,0 +1,198 @@
+}
+
+// Not really used as this is for --runall parameter, and we run cucumber step by step
+variable "CUCUMBER_COMMAND" {
+  type = "string"
+  default = "export PRODUCT='SUSE-Manager' && run-testsuite"
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = "string"
+  default = "https://github.com/SUSE/spacewalk.git"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "Manager-4.2"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = "string"
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = "string"
+  default = "Results 4.2-PRV $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = "string"
+  default = "Results 4.2-PRV: Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = "string"
+  default = "../mail_templates/mail-template-jenkins-env-fail.txt"
+}
+
+variable "MAIL_FROM" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = "string"
+  default = "galaxy-ci@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = "string"
+}
+
+variable "SCC_PASSWORD" {
+  type = "string"
+}
+
+variable "GIT_USER" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = "string"
+  default = null // Not needed for master, as it is public
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://metropolis.mgr.prv.suse.net/system"
+}
+
+
+module "cucumber_testsuite" {
+  source = "./modules/cucumber_testsuite"
+
+  product_version = "4.2-nightly"
+
+  // Cucumber repository configuration for the controller
+  git_username = var.GIT_USER
+  git_password = var.GIT_PASSWORD
+  git_repo     = var.CUCUMBER_GITREPO
+  branch       = var.CUCUMBER_BRANCH
+
+  cc_username = var.SCC_USER
+  cc_password = var.SCC_PASSWORD
+
+  images = ["centos7o", "opensuse152o", "sles15sp1o", "sles15sp2o", "sles15sp3o", "ubuntu2004o"]
+
+  use_avahi = false
+  name_prefix = "suma-42-"
+  domain = "mgr.prv.suse.net"
+  from_email = "root@suse.de"
+
+  no_auth_registry = "registry.mgr.prv.suse.net"
+  auth_registry = "registry.mgr.prv.suse.net:5000/cucutest"
+  auth_registry_username = "cucutest"
+  auth_registry_password = "cucusecret"
+  git_profiles_repo = "https://github.com/uyuni-project/uyuni.git#:testsuite/features/profiles/internal_prv"
+
+  mirror = "minima-mirror.mgr.prv.suse.net"
+  use_mirror_images = true
+  server_http_proxy = "galaxy-proxy.mgr.prv.suse.net:3128"
+
+  host_settings = {
+    controller = {
+      provider_settings = {
+        mac = "aa:b2:92:03:00:c0"
+      }
+    }
+    server = {
+      provider_settings = {
+        mac = "aa:b2:92:03:00:c1"
+      }
+      additional_repos = { java11 = "http://download.suse.de/ibs/home:/fstrba:/branches:/SUSE:/SLE-15:/Update/SUSE_SLE-15_Update/" }
```

Comment:
do we still want to test this java?

Reply:
No, that's a mistake. Seems my master was not synced with the upstream repo :-)

I will remove it.

Reply:
Fixed